### ISSUE TITLE
Not working on iOS

### ISSUE THREAD

**Issue by omersezer** [CrossGeeks/AppLinksSample#1]
works on android devices but I don't think it works on ios

**Comment by rdelrosario** [comment]
Yes it does work. Have been tested in both platform. You might have an iOS configuration missing if not working for you.

**Comment by omersezer** [comment]
I downloaded the project and then run but not working

**Comment by rdelrosario** [comment]
Did you do all iOS portal related configurations?

**Comment by rdelrosario** [comment]
Follow the steps on the blog post:http://www.xamboy.com/2019/01/08/applinks-in-xamarin-forms/

with your own iOS app.

**Comment by omersezer** [comment]
can you send as gif

**Comment by omersezer** [comment]
I did all of this, but it still didn't work

**Comment by keerthinathganjam** [comment]
yes not working on iOS, android its working
